airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/RedshiftDestination.java

```diff
@@ -58,6 +58,15 @@ public static DestinationType determineUploadMode(final JsonNode config) {
     return DestinationType.COPY_S3;
   }
 
+  @Override
+  public ConnectorSpecification spec() throws Exception {
```

Comment:
can this use the SshWrappedDestination class?

Reply:
No, it causes double wrapping of the endpoint addresses as the actual Insert and Staging Destinations internally are wrapped.